test/integration-tests/test-server/schema/schema.ts

```diff
@@ -0,0 +1,12 @@
+import { GraphQLSchema } from 'graphql';
+import { makeExecutablePolarisSchema } from '../../../../src/main';
+import { resolvers } from './resolvers/book-resolvers';
+import { Book, BookInput, Mutation, Query, Subscription } from './types/schema-types';
+
+export const schema: GraphQLSchema = makeExecutablePolarisSchema({
+    typeDefs: [Book, BookInput, Mutation, Query, Subscription],
+    resolvers: [resolvers],
+    resolverValidationOptions: {
+        requireResolversForResolveType: false,
```

Comment:
why?

Reply:
why not requireResolversForResolverType? do you have resolvers in schema that you didn't implement?